libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp

```diff
@@ -204,14 +204,25 @@ void AP_InertialSensor_Backend::_notify_new_gyro_raw_sample(uint8_t instance,
         _imu._last_delta_angle[instance] = delta_angle;
         _imu._last_raw_gyro[instance] = gyro;
 
+        // apply the low pass filter
         _imu._gyro_filtered[instance] = _imu._gyro_filter[instance].apply(gyro);
+        // apply the notch filter
+        if (_gyro_notch_enabled()) {
+            _imu._gyro_filtered[instance] = _imu._gyro_notch_filter[instance].apply(gyro);
```

Comment:
I think that here you need to use `_imu._gyro_filtered[instance]` instead of `gyro`:
```
_imu._gyro_filtered[instance] = _imu._gyro_notch_filter[instance].apply(_imu._gyro_filtered[instance]);
```
So you do not skip low pass filtering when notch is applied.

Reply:
Oh yes, good catch. Who needs a low pass filter anyway 🙂 I will fix. Very interesting though that you practically don't need the LPF if you have a notch enabled on these small copters.

Reply:
Maybe it's because I'm setting a very large bandwidth